setup/system.sh

```diff
@@ -144,6 +144,9 @@ pollinate  -q -r
 
 # Between these two, we really ought to be all set.
 
+echo 'Creating SSH public key... (might take a while)'
```

Comment:
For the file check, I'll add it.

For the key length, I like to keep my keys wide, as it's just making the time needed to break it a lot, lot, wider, so that in 5 years they're still good. And on any decent hardware, it's not too long to generate. BTW, I should actually add `-a 100`, though that makes it definitely longer to process.

Though to avoid blocking installation, we could run it in background, or even nohup, if it's too close to the end.